tests/csm/rest/test_csm_load.py

```diff
@@ -409,3 +409,51 @@ def test_32547(self):
                                                                    "statistics.json"))
         assert err_cnt == 0, f"{err_cnt} of {total_cnt} requests have failed."
         self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.lc
+    @pytest.mark.jmeter
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.tags('TEST-44782')
+    def test_44782(self):
+        """
+        Verify max allowed CSM user limit
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.test_cfgs["test_44782"]
+        jmx_file = "CSM_create_max_manage_users.jmx"
+        self.log.info("Running jmx script: %s", jmx_file)
+
+        request_usage = test_cfg["request_usage"]
+        total_users = test_cfg["total_users"]
+
+        loop = total_users // request_usage
```

Comment:
this wont work is request_usage is greater than total_users. Can u handle that scenario also.

Reply:
request_usage is 30 and total_users is 97 so calculation comes to be 90 users in 3 loops and 7 users in last loop